src/main/java/com/deavensoft/paketomat/user/UserService.java

```diff
@@ -10,23 +9,23 @@
 
```

Comment:
Please create UserSrevice as an interface. 
UserServiceImpl should implement UserService.